deps/db/src/logseq/db/sqlite/util.cljs

```diff
@@ -50,7 +50,11 @@
                                    "datascript/Entity" identity)
                             (merge read-handlers))
         reader (transit/reader :json {:handlers read-handlers*})]
-    (fn read-transit-str* [s] (transit/read reader s))))
+    (fn read-transit-str* [s]
+      (if (and (string? s) (identical? "[" (first s)))
```

Comment:
A comment for this check could be helpful. Unclear why we pass invalid calls through silently rather than fail explicitly

Reply:
The reason I didn't throw an error here is:  
If I accidentally missed deleting an ldb/read-transit-str in this PR, we can still see the :invalid-transit-string message in the console,  
without affecting the original normal logic.  

another reason for not using `assert` here:  
If assert is used here but ldb/read-transit-str is used within `p/let` (without explicit `p/catch`), I found no error log appears in the console, which provides no help in identifying the issue.

Reply:
Gotcha. Good to know it was useful for refactoring. Hopefully we can delete it later when there aren't any more missed deletions

Reply:
yes, we can keep it for maybe 1 month. let me add a todo in comment